packages/modules/imodel-browser/src/components/gridStructure/GridStructure.scss

```diff
@@ -10,11 +10,11 @@ $space-for-shadows: 3px;
   justify-content: space-between;
   grid-template-columns: repeat(auto-fill, minmax(288px, 1fr));
   grid-auto-rows: min-content;
-  margin: calc(22px - #{$space-for-shadows})
+  padding: calc(22px - #{$space-for-shadows})
     calc(#{$responsive-grid-margin} - #{$space-for-shadows});
   gap: $responsive-grid-gutter;
   overflow: hidden;
-  padding: $space-for-shadows;
+  margin: $space-for-shadows;
```

Comment:
Do we want to show the drop-shadow on three sides? The bottom and right shadow simulates a light-source from the top-left while a three-sided shadow would be impossible with a single light source. Might be worth verifying with UX to check for consistency.